types/deneb/blob_sidecar.yaml

```diff
@@ -6,83 +6,27 @@ Deneb:
     minItems: 0
     maxItems: 6
 
+  CommitmentInclusionProof:
```

Comment:
Given that we use `KZGCommitment` elsewhere, should we refer to this as `KZGCommitmentInclusionProof` or is this considered more generic?

Reply:
Updated field name to `kzg_commitment_inclusion_proof` https://github.com/ethereum/consensus-specs/pull/3531/commits/4a609cec574eca635a9e0980fe448089f8690ce1

Reply:
updated both the struct name and field name here: e9d053821a6e4dafe8651637322fb4ac4da01f6c